src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -276,15 +276,14 @@ private void recoverSubscriptionsOfChannel(int channelNumber) {
                     SubscriptionCallbacks subscriptionCallbacks = channelHolder.subscriptionCallbacks;
                     if (subscriptionCallbacks != null) {
                         channelsByPin.remove(pinId);
-                        if (channelHolder.isSubscribed) {
+                        if (channelHolder.subscribed()) {
                             LOGGER.info("Changing channel for holder with pin id: " + pinId.toString());
                             basicConsume(pinId.queue, subscriptionCallbacks.deliverCallback, subscriptionCallbacks.cancelCallback);
                         }
                     }
                 }
             } catch (IOException e) {
-                LOGGER.warn("Exception during channel's subscriptions recovery", e);
-                throw new RuntimeException(e);
+                LOGGER.warn("Failed to recovery channel's subscriptions", e);
```

Comment:
Why don't we throw an exception anymore? Could you please add a comment about why it is okay here?
Will we try to resubscribe again if it is a temporal error?